model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationRequestV1.java

```diff
@@ -91,6 +94,20 @@ public void setAction(final String action) {
         this.action = action;
     }
 
+    @ApiModelProperty(
+            value = "This list of policy set IDs specifies the order in which policy sets will be evaluated. "
+                    + "Evaluation stops when a policy with matching target is found and the condition returns true, "
+                    + "Or all policies are exhausted.")
```

Comment:
Can we use the active voice in these descriptions. E.g. "the order in which the service will evaluate policies".